Makefile

```diff
@@ -51,6 +53,12 @@ ifeq ($(cc_system_build),yes)
     QEMUBINDIR    := $(BINDIR)
     SYSCONFDIR    := /etc
     LOCALSTATEDIR := /var
+
+    ifeq (,$(installing))
```

Comment:
Forcing a clean for a non-install system build feels a little 'heavy handed', but I suspect any other better solution (such as removing, and thus forcing re-generation of, the generated file that holds the VERSION info) would require us holding some sort of 'status' file tracking that we've changed what sort of build we are doing.
Given the system-build steps are pretty well defined (build, and then install), I think this is fine as is. If we get a few extra clean/build cycles for people doing dev work, so be it.

Reply:
I agree. Yes, I guess we could just blat the generated file but I'm open to other ideas if anyone has any?